packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <p
+            :class="{
+                [$style['c-restaurantCard-iconText']]: true,
+                [$style['c-restaurantCard-iconText--bold']]: isBold
+            }"
+            aria-hidden="true"
+            :data-test-id="dataTestId">
```

Comment:
I can't see this being utilised in the DeliveryTimeMeta component - but like what I mentioned on the test file comment - would be worth passing something unique down here for eta vs distance/address and testing for that (sorry if you are but I can't see it!)

Reply:
I wrote some tests this morning that utilise that, merging them now 😄

Reply:
Alternatively, I can apply the test id to the parent icon-text element and not pass it down as a prop

Reply:
tbh I quite like the idea of just applying it from the parent component as opposed to a prop - the prop is super useful if we need granular control over any nested test IDs - so happy to go with your preference on this